crates/wasmtime/src/runtime/component/instance.rs

```diff
@@ -55,7 +55,7 @@ impl Instance {
     ///
     /// The safety of this function relies on `id` belonging to the instance
     /// within `store`.
-    pub(crate) unsafe fn from_wasmtime(store: &StoreOpaque, id: ComponentInstanceId) -> Instance {
+    pub(crate) fn from_wasmtime(store: &StoreOpaque, id: ComponentInstanceId) -> Instance {
```

Comment:
Is the sentence regarding safety in the doc comment above still relevant, or should it be removed?